src/python/WMComponent/RucioInjector/RucioInjectorPoller.py

```diff
@@ -268,12 +268,13 @@ def insertBlockRules(self):
             if not self._isBlockTierAllowed(item['blockname']):
                 logging.debug("Component configured to skip block rule for: %s", item['blockname'])
                 continue
+            kwargs = dict(activity="Production Output", account=self.rucioAcct,
+                          grouping="DATASET", comment="WMAgent automatic container rule",
+                          meta=self.metaData)
             rseName = "%s_Test" % item['pnn'] if self.testRSEs else item['pnn']
             # DATASET = replicates all files in the same block to the same RSE
-            resp = self.rucio.createReplicationRule(item['blockname'], rseExpression="rse=%s" % rseName,
-                                                    account=self.rucioAcct, grouping="DATASET",
-                                                    comment="WMAgent production site",
-                                                    meta=self.metaData)
+            resp = self.rucio.createReplicationRule(item['blockname'],
+                                                    rseExpression="rse=%s" % rseName, **kwargs)
```

Comment:
So this is a matter of taste, but `rse={rseName}` can also be written simply `{rseName}`, that is, the RSE expression can be also a single RSE name. Whatever you choose, as long as you stick to it I don't think there will be much issue.

Reply:
Personally I'm in favor of the single-name, but I know for example that @ericvaandering 's sync script also uses `rse=` style.

Reply:
I'd rather use a single-name too. I guess I tried in the past with a basic string and that failed, then I used the same way Eric uses in his script. It's good to know it could be a set. I guess I'll leave it untouched for now.

Reply:
Basic string should work, I've definitely used it, e.g. in this gist: https://gist.github.com/nsmith-/3074cadda71a91ae60ebe16039cfa1dc#file-rucio_outputdataset-py-L54